test/HistoryGTest.cpp

```diff
@@ -0,0 +1,77 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file HistoryGTest.cpp
+* @brief unit tests for history functions
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include <cmath>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```

Comment:
gmock and gtest should be down next to the flatbuffer one

Reply:
is that because it's not a built in thing? I was always going off the carrots  <>

Reply:
you can use angle brackets and quotes interchangeably.

It is legal to do `#include <History.h>` or `#include "iostream"`

Dune coding style says you should only use angle brackets <> for system libraries. gmock and gtest should technically be surrounded with quotes. It isn't uncommon to see them with angle brackets in dune code though, I think it is just because that is the way it is in the gtest documentation.